### PR TITLE
Add replication_pool_map to disaster recovery data sources

### PR DESCRIPTION
**Pull request by Alexander-Kita**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

pi_disaster_recovery_location:

```
=== RUN   TestAccIBMPIDisasterRecoveryLocationDataSourceBasic
--- PASS: TestAccIBMPIDisasterRecoveryLocationDataSourceBasic (5.74s)
PASS
```

pi_disaster_recovery_locations:

```
=== RUN   TestAccIBMPIDisasterRecoveryLocationsDataSourceBasic
--- PASS: TestAccIBMPIDisasterRecoveryLocationsDataSourceBasic (6.35s)
PASS
```

Description: Added replication_pool_map attribute to pi_disaster_recovery_location and  pi_disaster_recovery_locations datasources.

### CODE REVIEW

File: website/docs/d/pi_disaster_recovery_location.html.markdown [hunk 1]
```diff
@@ -46,3 +46,7 @@ In addition to all argument reference list, you can access the following attribu
   Nested scheme for `replication_sites`:
   - `is_active` - (Boolean) Indicates the location is active or not, `true` if location is active , otherwise it is `false`.
   - `location` - (String) The region zone of the location.
+  - `replication_pool_map` - (List) List of maps containing information about volume and replica pools
```
Comment (michaelkad) [inline website/docs/d/pi_disaster_recovery_location.html.markdown:49]:
```suggestion
  - `replication_pool_map` - (List) List of replication pool map.
```

File: website/docs/d/pi_disaster_recovery_location.html.markdown [hunk 2]
```diff
@@ -46,3 +46,7 @@ In addition to all argument reference list, you can access the following attribu
...
+    Nested scheme for `replication_pool_map`:
+    - `remote_pool` - (String) Pool where remote volume is located
+    - `volume_pool` - (String) Pool where main volume is located
```
Comment (michaelkad) [inline website/docs/d/pi_disaster_recovery_location.html.markdown:52]:
```suggestion
    - `remote_pool` - (String) Remote pool.
    - `volume_pool` - (String) Volume pool.
```

File: website/docs/d/pi_disaster_recovery_locations.html.markdown [hunk 3]
```diff
@@ -38,7 +38,10 @@ In addition to all argument reference list, you can access the following attribu
...
   - `location` - (String) The region zone of a site.
   - `replication_sites` - List of Replication Sites.
-  
         Nested scheme for `replication_sites`:
         - `is_active` - (Boolean) Indicates the location is active or not, `true` if location is active, otherwise it is `false`.
         - `location` - (String) The region zone of the location.
+        - `replication_pool_map` - (List) List of maps containing information about volume and replica pools
+          Nested scheme for `replication_pool_map`:
+          - `remote_pool` - (String) Pool where remote volume is located
+          - `volume_pool` - (String) Pool where main volume is located
```
Comment (michaelkad) [inline website/docs/d/pi_disaster_recovery_locations.html.markdown:47]: Same as above

**Review by michaelkad — CHANGES_REQUESTED** [review]
I have left few comments that need to be addressed.

**Review by michaelkad — APPROVED** [review]
Lgtm

**Review by ismirlia — APPROVED** [review]

**Pull request closed without merging**